.github/workflows/codecov-upstream.yml

```
@@ -0,0 +1,18 @@
+# Documentation: https://docs.codecov.io/docs/codecov-yaml
+
+codecov:
+  # Avoid "Missing base report" due to committing CHANGES.rst with "[CI skip]"
+  # https://github.com/codecov/support/issues/363
+  # https://docs.codecov.io/v4.3.6/docs/comparing-commits
```

```suggestion
  # https://docs.codecov.io/docs/comparing-commits
```
Fixes the 404 as per #4267